### DISCUSSION TITLE
Where are 3.10 release notes

### DISCUSSION THREAD

**Discussion by nickdring** [joomla/joomla-cms#35217] [General] [answered]
Hi, I cannot find any release notes for 3.10. I've read this https://www.joomla.org/announcements/release-news/5845-joomla-4-0-and-joomla-3-10-are-here.html but it doesn't say if this ONLY for J4 migration or if its for anything else such as bug fixes and security patches. I don't intend on going to J4 anytime soon with our sites so I'm wondering if I need to install 3.10 or not. Hence my question.
Thanks

**Comment by brianteeman** [comment, marked as answer]
Hi Nick

There are no security fixes in 3.10.0
Any future security or bug fixes in the Joomla 3 series will begin in 3.10.1
There will be no further updates to 3.9
You can really think of 3.10 and 3.9.29
3.10 will be supported until 17 Aug 2023

**Reply by nickdring** [reply]
Thanks Brian!
